build-system/tasks/presubmit-checks.js

```diff
@@ -421,6 +421,9 @@ const forbiddenTerms = {
   'cidForDoc|cidForDocOrNull': {
     message: requiresReviewPrivacy,
     whitelist: [
+      // CID service is not allowed in amp4ads. No usage should there be
```

Comment:
Is `'src/service/url-replacements-impl.js',` an exception? It's very possible that amp4ads use some other service which then use cid service. But I guess that's fine, because we reject the cid service in amp4ads anyway.

Reply:
Yep, we reject CID service in anyway. This is to add another layer of sanity check to make sure any new usage of those 2 APIs should give some thoughts into ads use cases, e.g. provide a fallback or so.